test/test_dataclass_validations.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+
+from openwpm.config import validate_browser_params, validate_manager_params
+from openwpm.errors import ConfigError
+
+from .openwpmtest import OpenWPMTest
+
+
+class TestDataclassValidations(OpenWPMTest):
```

Comment:
OpenWPMTest is only useful, if you actually want to start up a TaskManager.
I think it would be best to turn these methods into functions similiar to the [test_js_instrument_py.py](https://github.com/mozilla/OpenWPM/blob/master/test/test_js_instrument_py.py)
As long as the functions still start with `test_` pytest will recognize them